text/docs-code-contributions.md

```diff
@@ -0,0 +1,70 @@
+- Start Date: 2019-02-07)
+- RFC PR: 
+- Gatsby Issue:
+
+# Summary
+
+Reorganize the contributor documentation to aid in Gatsby core and docs development.
+
+# Motivation
+
+The Gatsby.js docs, while wonderful, are light on coding instructions for contributors. With dedicated sections in the Contributing docs, it will be easier to learn how to clone the repository, where to find certain things like docs or plugins, how to run tests, and insider infrastructure knowledge required to submit a successful pull request on the first try.
+
+A lot of this information already lives on the "How to Contribute" page, but it's buried. It would help discoverability if this information lived on separate pages.
+
+# Detailed design
+
+I propose that we add some pages to the existing contributor documentation and move some content around. Here is an IA diagram that (briefly) reflects the current structure and what I think we should add (open for discussion, of course):
+
+Docs
+- Guides
+- Ecosystem
+- API Reference
+- Releases & Migration
+- Conceptual Guide
+- Behind the Scenes
+- Advanced Tutorials
+
+Contributing
+- Community
+    - Why Contribute to Gatsby?
+    - Pair Programming Sessions
+    - Free Swag for Contributors
+    - Where to Participate in the Community
+    - How to Run a Gatsby Workshop
+    - How to Pitch Gatsby
+- Code of Conduct
+- Gatsby Style Guide
+- How to Contribute
+    * How to File an Issue
+    * How to Label an Issue
+    * Managing Issues and Pull Requests
```

Comment:
Following on from https://github.com/gatsbyjs/rfcs/pull/29#pullrequestreview-203382007

Could we split this out to _Triaging issues_ and _Reviewing Pull Requests_? I think they're different enough for that to be useful. @sidharthachatterjee has been working on an issue triage doc which could easily slot in here. (Sid could you post a link to your WIP doc?)

For now we could skip creating a _Reviewing Pull Requests_ stub, then come back and add it once we have content for it.

Reply:
That structure sounds great to me! @sidharthachatterjee I would love to see your issue triage doc. 🙌